osgp/shared/osgp-ws-smartmetering/src/main/resources/schemas/sm-installation.xsd

```diff
@@ -159,6 +159,51 @@
     </xsd:complexContent>
   </xsd:complexType>
 
+  <xsd:element name="DeCoupleMbusDeviceByChannelRequest">
+    <xsd:complexType>
+      <xsd:sequence>
+        <xsd:element name="DeviceIdentification" type="common:DeviceIdentification" />
+        <xsd:element name="DeCoupleMbusDeviceByChannelRequestData"
+                     type="tns:DeCoupleMbusDeviceByChannelRequestData" />
+      </xsd:sequence>
+    </xsd:complexType>
+  </xsd:element>
+
+  <xsd:element name="DeCoupleMbusDeviceByChannelAsyncResponse">
+    <xsd:complexType>
+      <xsd:complexContent>
+        <xsd:extension base="common:AsyncResponse"></xsd:extension>
+      </xsd:complexContent>
+    </xsd:complexType>
+  </xsd:element>
+
+  <xsd:element name="DeCoupleMbusDeviceByChannelAsyncRequest">
+    <xsd:complexType>
+      <xsd:complexContent>
+        <xsd:extension base="common:AsyncRequest"></xsd:extension>
+      </xsd:complexContent>
+    </xsd:complexType>
+  </xsd:element>
+
+  <xsd:element name="DeCoupleMbusDeviceByChannelResponse">
+    <xsd:complexType>
+      <xsd:complexContent>
+        <xsd:extension base="tns:DeCoupleMbusDeviceByChannelResponseData">
+        </xsd:extension>
+      </xsd:complexContent>
+    </xsd:complexType>
+  </xsd:element>
+
+  <xsd:complexType name="DeCoupleMbusDeviceByChannelResponseData">
+    <xsd:complexContent>
+      <xsd:extension base="common:Response">
+        <xsd:sequence>
+          <xsd:element name="mbusDeviceIdentification" type="common:DeviceIdentification"/>
```

Comment:
If I am not mistaken there are test cases where the M-Bus device identification in the decouple M-Bus device by channel response is not set. It looks like such a response would be invalid, and a `minOccurs = "0"` is missing.

Reply:
Correct M-Bus device identification might not be set therefor added `minOccurs = "0"` to the `sequence`

Reply:
Have to check this with @kroesctrl

Reply:
checked